### PR TITLE
use dedent for config/get_contexts.go

### PR DESCRIPTION
**Pull request by lojies**
use dedent for config/get_contexts.go

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/32109)

### CODE REVIEW

**Comment by dims** [comment]
@k8s-ci-robot ok to test

**Comment by dims** [comment]
ok to test

**Comment by dims** [comment]
ok to test

**Comment by lojies** [comment]
@pwittrock @dims

**Comment by lojies** [comment]
@pwittrock can this be test

**Comment by lojies** [comment]
@AdoHe

**Comment by lojies** [comment]
@AdoHe can this be test？

File: pkg/kubectl/cmd/config/get_contexts.go [hunk 1]
```diff
@@ -41,14 +42,16 @@ type GetContextsOptions struct {
...
-kubectl config get-contexts
+	getContextsExample = dedent.Dedent(`
+		# List all the contexts in your kubeconfig file
+		kubectl config get-contexts
 
-# Describe one context in your kubeconfig file.
-kubectl config get-contexts my-context`
+		# Describe one context in your kubeconfig file.
+		kubectl config get-contexts my-context
+	`)
```
Comment (AdoHe) [inline pkg/kubectl/cmd/config/get_contexts.go]: no need to add newline here.

**Comment by AdoHe** [comment]
@lojies just one comment

**Comment by AdoHe** [comment]
ok to test

**Comment by lojies** [comment]
@k8s-bot gci gce e2e test this

**Comment by AdoHe** [comment]
@lojies we are going to replace dedent with our own normalizers, you may take a look #34502

**Comment by lojies** [comment]
@AdoHe ok, this file has been replaced in https://github.com/kubernetes/kubernetes/pull/34502 and i will close this pr.

**Pull request closed without merging**